windows/src/engine/keyman32/Keyman32.cpp

```diff
@@ -310,8 +311,10 @@ BOOL InitialiseProcess(HWND hwnd)
 	wm_keymanim_contextchanged = RegisterWindowMessage("WM_KEYMANIM_CONTEXTCHANGED");
 	wm_keymanshift = RegisterWindowMessage("WM_KEYMANSHIFT");
 
+
 	wm_keyman_control = RegisterWindowMessage(RWM_KEYMAN_CONTROL);
 	wm_keyman_control_internal = RegisterWindowMessage("WM_KEYMAN_CONTROL_INTERNAL");
+  wm_keyman_modifierevent = RegisterWindowMessage("WM_KEYMAN_MODIFIEREVENT");
```

Comment:
We should be able to do this and `wm_keyman_keyevent` as simple WM_USER messages, rather than globally registered messages, because they are only ever used internally to a window we own. But I think that change belongs in its own maintenance PR.

Reply:
created #7458